collection/rancher/v2.x/logs-collector/rancher2_logs_collector.sh

```diff
@@ -541,18 +545,33 @@ cleanup() {
 
 }
 
+upload() {
+
+  if $(command -v curl >/dev/null 2>&1); then
+    curl -v -L -A SupportConfig \
+	    -T "${FILEDIR}/${FILENAME}.gz" \
+	    "${SUSE_UPLOAD_HTTPS}/upload.php?appname=supportconfig&file=${FILENAME}.gz"
+  else
+    techo "Upload requires curl to be present."
```

Comment:
Good idea to check for curl and other prerequisites before generating the support bundle.